go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -265,19 +268,34 @@ func (m *ModelClient) buildOps(doWhenFound opModelToOpMapper, doWhenNotFound opM
 /*
  create does a bit more than just "create". create needs to set the generated
  UUID (because if this function is called we know the item does not exists yet)
- then create the item.
+ then create the item. Generates an until clause and uses a wait operation to avoid
+ https://bugzilla.redhat.com/show_bug.cgi?id=2042001
 */
 func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error) {
 	uuid := getUUID(opModel.Model)
 	if uuid == "" {
 		setUUID(opModel.Model, BuildNamedUUID())
 	}
+	ops := make([]ovsdb.Operation, 0)
+	if len(opModel.Name) > 0 {
+		ops = append(ops, ovsdb.Operation{
+			Op:      "wait",
+			Timeout: 0,
+			Table:   "Load_Balancer",
```

Comment:
heh, say what table this is, again?!?

Reply:
oops as you can tell i was quite tired :)